src/config/endpoints.ts

```diff
@@ -2,3 +2,4 @@ export const SIGNIN = "/api/sign-in";
 export const SIGNUP = "/api/sign-up";
 export const GET_USER_REPOSITORIES = "/api/repo";
 export const GET_CHALLENGES = "/api/challenge";
+export const CREATE_REPO = "/api/repo";
```

Comment:
To avoid confusion we should have e.g. REPO_ENDPOINT, CHALLENGES_ENDPOINT etc.  As right now the CREATE_REPO & GET_USER_REPOSITORIES constants are the same yet have different names.  Just a suggestion and can be fixed later if you rather get this merged.